share/spice/crunchbase/collections/crunchbase_collections.js

```diff
@@ -0,0 +1,42 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_crunchbase_collections = function(api_result){
+        // Validate the response
+        if (!api_result || api_result.error || api_result.data.items.length === 0) {
```

Comment:
Check if `api_result.data` and `api_result.data.items` exists before accessing it's length property just in case they don't exist.